### ISSUE TITLE
Empty frames while recording frames with live capture

### ISSUE THREAD

**Issue by GuLinux** [qhyccd-lzr/QHYCCD_Linux#14]
Hello,
this issue is a bit difficult to reproduce, I'm able to reproduce it only on a netbook with an Atom CPU and 2 GB of ram, so I guess it's related to poor hardware condtions.

When I start a capture, for the first frames everything is ok, but after a while (full memory buffers maybe? here it's after about 1000 saved frames) the driver starts acting strange.
GetQHYCCDLiveFrame always exits with QHYCCD_SUCCESS, but it doesn't wait for the frame being correctly downloaded, and the buffer is filled with zeroes.

So basically what I get is a higher framerate (much higher than it should be, like 100/200 fps), and a lot of empty frames.

As a workaround, I added a check on my application code, checking for frames filled with zeroes (see here: https://github.com/GuLinux/PlanetaryImager/commit/0c8862e7db70b0fc7efa06a984d4c3edf786d98f ) . If I skip those frames, the application starts to work properly.

**Comment by GuLinux** [comment]
a related sidenote: if I pull out the cable while the camera is capturing, GetQHYCCDLiveFrame still returns 0 (QHYCCD_SUCCESS).
